.gitignore

```diff
@@ -10,6 +10,9 @@
 # Compiled python code.
 *.pyc
 
+# Distribution / packaging
+*.egg-info/
```

Comment:
Will skip the make targets for now, when we start really packaging robottelo then we add them as we will need refactor the Makefile.